src/components/Advertisements/core/AdvertisementRegister/AdvertisementRegister.tsx

```
@@ -428,57 +411,51 @@
-                          label="English captions"
+
+                {(formState.attachments || []).map(
+                  (file: any, index: number) => (
+                    <div key={index}>
+                      {file.localFile?.type.startsWith('video/') ? (
+                        <video
+                          data-testid="mediaPreview"
+                          controls
+                          src={URL.createObjectURL(file.localFile)}
+                          className={styles.previewAdvertisementRegister}
                         />
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)